search-app/src/cmr/search/api/request_context_user_augmenter.clj

```diff
@@ -38,12 +41,11 @@
   (let [{:keys [token]} context]
     (let [sids (cache/get-value (cache/context->cache context token-sid-cache-name)
                                 token
-                                #(or (sids-retriever/get-sids context token)
-                                     {:sids (acl/context->sids context)}))
-          token-guid (:guid sids)
```

Comment:
I see - so we were printing the token guid to avoid printing the actual token?  not sure how to get around that without adding an extra call to get the guid somewhere (and caching it)